projects/plugins/jetpack/modules/likes.php

```diff
@@ -432,9 +432,10 @@ function_exists( '\Activitypub\is_activitypub_request' )
 		* If the same post appears more then once on a page the page goes crazy
 		* we need a slightly more unique id / name for the widget wrapper.
 		*/
-		$uniqid = uniqid();
+		$uniqid     = uniqid();
+		$new_layout = apply_filters( 'likes_new_layout', false ) ? '&amp;n=1' : '';
```

Comment:
Could you add a docblock here, so the new hook can be documented in the codex?

```suggestion
		/**
		 * Enable an alternate Likes layout.
		 *
		 * @since $$next-version$$
		 *
		 * @module likes
		 *
		 * @param bool $new_layout Enable the new Likes layout. False by default.
		 */
		$new_layout = apply_filters( 'likes_new_layout', false ) ? '&amp;n=1' : '';
```